ui/selectors/selectors.js

```diff
@@ -145,7 +145,7 @@ import { getSelectedMultichainNetworkChainId } from './multichain/networks';
 import { getRemoteFeatureFlags } from './remote-feature-flags';
 import { getApprovalRequestsByType } from './approvals';
 
-export const isGlobalNetworkSelectorRemoved = process.env.REMOVE_GNS;
+export const isGlobalNetworkSelectorRemoved = true;
```

Comment:
Oooh nice, is this intentional for this PR?

Reply:
Ah fair enough. My 2 concerns are:
1. Environment variable complexity - worried that the more variables we add, the more things we need to manage. This might be okay if GNS will be removed soon.
2. Worst case being able to "revert" these changes.

I think we can go with number 3 as long as the number of variables we are managing doesn't get too complex.